terraform/scripts/foundation.sh

```diff
@@ -5,11 +5,26 @@
 
 set -eux
 
-sudo mount /dev/xvdf /mnt
-echo '/dev/xvdf /mnt     ext4   defaults 0 0' | sudo tee -a /etc/fstab
+# Set up the filesystem
+sudo mkfs.ext4 /dev/nvme1n1
+sudo mount /dev/nvme1n1 /mnt
+echo '/dev/nvme1n1 /mnt     ext4   defaults 0 0' | sudo tee -a /etc/fstab
```

Comment:
Is `nvme1n1` consistent across instance types? Is there a way we can influence or tag that disk via terraform?

Reply:
no, this is defined by AWS